go/build-and-test/action.yml

```diff
@@ -32,6 +32,10 @@ inputs:
   TEST_TAGS:
     description: Test tags
     required: false
+  TIMEOUT:
+    description: A timeout value in seconds. If tests take longer than this, they will fail. Set to an empty string to disable.
```

Comment:
Wait, is this true? If the default input is an empty string, and an empty string will disable it, does that mean there's no longer a default timeout of 10 minutes?

... or was the default previously 30 seconds? Now I'm confused.

Reply:
The script (and action) had it as 30s. The default for the `go test` command itself is 10m. I set the default to blank (which won't set a timeout value in the script) and so it'll default to 10m.

I should actually change the wording to reflect that an empty string will default the timeout to 10m.

Reply:
Ah - I was confused when you said "keeping the default" - we are actually _changing_ the default for this action from 30s to 10m.

Wording needs updating yeah - if I understand what you said, setting it to an empty string won't actually disable the timeout entirely, it'll just use the default of 10m, right?

Reply:
Yep that's right. It causes the timeout option not to be added to the `go test` command and the `go test` command will timeout after 10 minutes.